spec/requests/api/chargebacks_assignement_spec.rb

```diff
@@ -0,0 +1,71 @@
+RSpec.describe "chargebacks API" do
+  it "can fetch the list of all tags of one chargeback" do
+    # Build Chargeback tag
+    chargeback = FactoryGirl.create(:chargeback_rate)
+    tenant     = FactoryGirl.create(:tenant)
+    ChargebackRate.set_assignments chargeback.rate_type, [{:cb_rate => chargeback, :object => tenant}]
+    # URI of tags
+    uri = "#{chargebacks_url(chargeback.id)}/tags"
+    tag = chargeback.tags.first
+
+    api_basic_authorize action_identifier(:chargebacks, :read, :resource_actions, :get)
+    run_get uri, :expand => "resources"
+    expect_result_to_match_hash(
+      response_hash,
```

Comment:
`response_hash` has now been removed - you'll need to use `response.parsed_body` instead